Sources/Sentry/SentryUIEventTracker.m

```diff
@@ -0,0 +1,166 @@
+                    [self.activeTransaction dispatchIdleTimeout];
+                    return;
+                }
+
+                [self.activeTransaction finish];
+
+                NSString *operation = [self getOperation:sender];
+
+                SentryTransactionContext *context =
+                    [[SentryTransactionContext alloc] initWithName:transactionName
+                                                         operation:operation];
+
+                BOOL ongoingScreenLoadTransaction = span != nil &&
+                    [span.context.operation isEqualToString:SentrySpanOperationUILoad];
+                BOOL ongoingManualTransaction = span != nil
+                    && ![span.context.operation isEqualToString:SentrySpanOperationUILoad]
+                    && ![span.context.operation containsString:SentrySpanOperationUIAction];
+
+                BOOL bindToScope = !ongoingScreenLoadTransaction && !ongoingManualTransaction;
+                SentryTracer *transaction =
+                    [SentrySDK.currentHub startTransactionWithContext:context
+                                                          bindToScope:bindToScope
+                                                customSamplingContext:@{}
+                                                          idleTimeout:self.idleTimeout
+                                                 dispatchQueueWrapper:self.dispatchQueueWrapper];
+
+                if ([[sender class] isSubclassOfClass:[UIView class]]) {
+                    UIView *view = sender;
+                    if (view.accessibilityIdentifier) {
+                        [transaction setTagValue:view.accessibilityIdentifier
+                                          forKey:@"accessibilityIdentifier"];
+                    }
+                }
+
+                transaction.finishCallback = ^(void) { self.activeTransaction = nil; };
+
+                self.activeTransaction = transaction;
+            }];
+        }
+                   forKey:SentryUIEventTrackerSwizzleSendAction];
+}
+
+- (void)stop
+{
+    [self.swizzleWrapper removeSwizzleSendActionForKey:SentryUIEventTrackerSwizzleSendAction];
+}
+
+- (NSString *)getOperation:(id)sender
+{
+    Class senderClass = [sender class];
+    if ([senderClass isSubclassOfClass:[UIButton class]] ||
+        [senderClass isSubclassOfClass:[UIBarButtonItem class]] ||
+        [senderClass isSubclassOfClass:[UISegmentedControl class]] ||
+        [senderClass isSubclassOfClass:[UIPageControl class]]) {
+        return SentrySpanOperationUIActionClick;
+    }
+
+    return SentrySpanOperationUIAction;
+}
+
+/**
+ * The action is an Objective-C selector and might look weird for Swift developers. Therefore we
+ * convert the selector to a Swift appropriate format aligned with the Swift #selector syntax.
+ * method:first:second:third: gets converted to method(first:second:third:)
+ */
+- (NSString *)getTransactionName:(NSString *)action target:(id)target
```

Comment:
Here are some usage samples to simplify my explanation:

Swift Function | Result | How I think it should be to not cause confusion
- func testClick() |  iOS_Swift.ViewController.testClick | iOS_Swift.ViewController.testClick()
- func testClick(_ sender : Any) | iOS_Swift.ViewController.testClick | iOS_Swift.ViewController.testClick(_:)
- func testClick(sender : Any) | iOS_Swift.ViewController.testClickWithSender | iOS_Swift.ViewController.testClick(sender:)
- func testClick(sender : Any, second : Any) | iOS_Swift.ViewController.testClickWithSender(second:)  | iOS_Swift.ViewController.testClick(sender:second:)
- func testClick(_ sender : Any, second : Any) | iOS_Swift.ViewController.testClick(second:) | iOS_Swift.ViewController.testClick(_:secondParam:)

As I said before, it is a little trickier to achieve perfect solution, Im kind fine with the result we have right now, because the first and second itens are more common, but I do believe this will cause some confusion with some users.

Reply:
Hmm, I get your point but aren't most of these events coming from the storyboard, and there we only have 

- `func testClick()`
- `func testClick(_ sender : Any)`
- `func testClick(_ sender : Any, _ second : Any)`

I agree that our solution is not optimal, but I think it's enough for you to figure out which code was executed. The problem is that only when looking at the action I don't know how to get the difference between `func testClick(_ sender : Any)` and `func testClick()` without using other objc_runtime functions. Do have any idea how we could solve this? Maybe we can also fix this before enabling `enableUserInteractionTracing` per default.

Reply:
Are you fine with creating an issue for this and fix it later @brustolin?

Reply:
> I don't know how to get the difference between func testClick(_ sender : Any) and func testClick()

if actions contain `:`, it has parameter, otherwise not.
But I thing we can release this way and wait for feedbacks